README.md

```diff
@@ -39,6 +39,7 @@ launching a Kubernetes cluster hosted on AWS.
 * Ability to generate configuration files for AWS [CloudFormation](https://aws.amazon.com/cloudformation/) and Terraform [Terraform configuration](/docs/terraform.md)
 * Supports custom Kubernetes [add-ons](/docs/addons.md)
 * Command line [autocompletion](/docs/cli/kops_completion.md)
+* Manifest Based API [Configuration](/docs/manifests_and_customizing_via_api.md)
```

Comment:
Is "manifest" now the term we're standardizing on? I actually had to look that word up but I don't really care as long as there's one single word to call "the YAML stuff".